classes/options-general.php

```diff
@@ -289,7 +289,7 @@ function print_page() {
 
 													<tr valign="top">
 														<td width="150"><label
-																for="default_country"><?php _e( 'Default Country', 'simplemap' ); ?></label>
+																for="default_country"><?php _e( 'Default Country', 'SimpleMap' ); ?></label>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 140 characters

<!-- Comment by [SideCI](https://sideci.com) -->